src/core.js

```diff
@@ -83,7 +83,7 @@ jQuery.fn = jQuery.prototype = {
 	},
 
 	// Execute a callback for every element in the matched set.
-	each: function( callback ) {
+	each:function(callback ) {
```

Comment:
format: style mismatch:
```suggestion
	each: function( callback ) {
```

`489e0fea` ␣ should be inserted at column 7.
`55a8147f` ␣ should be inserted at column 16.